src/streaming-services/vrtnu-be/VrtnuBeApi.ts

```diff
@@ -0,0 +1,24 @@
+import { Api } from '../common/Api';
+import { registerApi } from '../common/common';
+
+// Define any types you need here.
```

Comment:
These comment are really just for the template, and doesn't make so much sense here.

Reply:
I've already merged the other PRs, which also had these comments, so I can just remove them myself later when I work on the general cleaning PR.

Reply:
Oh sorry. I didn't really check those files anymore and indeed just copy/paste to the other branches